packages/react-dom/src/__tests__/ReactErrorBoundaries-test.js

```diff
@@ -819,7 +819,7 @@ describe('ReactErrorBoundaries', () => {
     expect(log).toEqual(['ErrorBoundary componentWillUnmount']);
   });
 
-  it('propagates errors inside boundary during componentWillMount', () => {
+  fit('propagates errors inside boundary during componentWillMount', () => {
```

Comment:
Fiit

Reply:
This is so easy to do. Maybe we should add an ESLint rule against it.

Reply:
Yes please